ig.json

```diff
@@ -1,7 +1,7 @@
 {
   "paths": {
     "resources": ["resources"],
-    "specification": "http://hl7.org/fhir/smart-app-launch",
+    "specification": "http://hl7.org/fhir/STU3/",
```

Comment:
http://wiki.hl7.org/index.php?title=IG_Publisher_Documentation shows this `specification` property without a trailing slash.

Reply:
good catch - both work but agree this is how documented and should not end with the slash.